MyHR/moment-business-time
# 3976-dependency-updates

## In this PR

- Upgrade node version from 8 to 14
- Update dependencies so there are no remaining vulnerabilities

## References

- [ ] resolves MyHR/issues#3976
- [ ] api-hubspot PR: https://github.com/MyHR/api-hubspot/pull/761

## Testing

- [x] Project tests passing?

### To manually test
1. Deploy a new version of the package
2. In api-hubspot or api-documents, update the version of @myhr/moment-business-time in package.json and npm install
3. Request a new document in your local environment and check when it is due
4. Change your local day/time to be close to the weekend and request another new document and check that it has takent he weekend into account
5. Change your local day/time to be the day before a national NZ public holiday and request another new document and check that it has taken the public holiday into account

## Base branch

- [x] Is this branched off `master`?
- [ ] Another branch: `branch name here`

## Deployment

- [ ] once this PR is merged and a new version of the package deployed, `api-hubspot` and `api-documents` should be updated to use the new version

## Release note

  - [ ] Reason: dependency updates

## Additional

- [Bringing a healthy code review mindset](https://www.smashingmagazine.com/2019/06/bringing-healthy-code-review-mindset/)
